plugin_tooling/src/com/github/rustdt/tooling/RustBuildOutputParser.java

```diff
@@ -1,21 +1,20 @@
 /*******************************************************************************
```

Comment:
Differences in this file are because I separated the regexp-based code out of this file, so that I could re-use the regexp-based code for the cargo output.